src/background/listener/bee-api.listener.ts

```diff
@@ -149,7 +149,7 @@ export class BeeApiListener {
         const bzzReference = subdomainToBzzResource(subdomain) + pathWithParams
         console.log('bzz link redirect', bzzReference, url, pathWithParams)
 
-        this.redirectToBzzReference(bzzReference, tabId)
+        this.redirectToBzzReference(bzzReference, tabId, true)
```

Comment:
why do you do that? `cid.bzz.link` also should be mapped to `{cid}.swarm.localhost`